projects/canopy/src/styles/variables/components/_side-nav.scss

```diff
@@ -0,0 +1,4 @@
+:root {
+  --side-nav-bar-item-border-width-md: 0.1875rem; // 3px
```

Comment:
you could probably link this up to this variable, I think the intention is the same. It's the one we use on the left border for a selected input field.
```suggestion
  --side-nav-bar-item-border-width-md: var(--keyline-width);

Reply:
hmm, thats a maybe, as there currently isn't a `var(--keyline-width-lg);` for the other one, so it could get confusing.

Reply:
I'll leave this as it is for now, if that's ok with you.